### PR TITLE
TR-894: Add rtl for order lists

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/TR-894
3/3

Required
https://github.com/oat-sa/tao-core/pull/3096

### CODE REVIEW

**Review by marpesia — CHANGES_REQUESTED** [review]
It could be great to have some info about the changes and How to test them in the PR description box.
Besides a composer.json file, I've seen yours is different from mine.

File: views/scss/qti/_qti-base.scss [hunk 1]
```diff
@@ -218,8 +220,14 @@ $mainContainer : '.qti-item';
         }
         &.solid, &.bordered {
             padding-top: 3px;
-            padding-right: 3px;
+            // padding-right: 3px;
```
[the remark this answers is not in the document]
Comment (meefox) [reply views/scss/qti/_qti-base.scss:223]: oh may, thanks

**Comment by meefox** [comment]
> It could be great to have some info about the changes and How to test them in the PR description box.
> Besides a composer.json file, I've seen yours is different from mine.

yes, it is one the way :) coming together with test env.

**Review by marpesia — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by jsconan — CHANGES_REQUESTED** [review]
The code looks ok.
However, there is an issue when mixing RTL layout with LTR item: the LTR item is displayed in RTL.
image

**Comment by meefox** [comment]
> The code looks ok.
> However, there is an issue when mixing RTL layout with LTR item: the LTR item is displayed in RTL.
> image

Added rule to be dependant on item's level dir RTL/LTR.
This kind of problem will persist till we add an item level `dir= LTR` to override `dir=RTL` from `body` in order to set DOM to LTR direction.

**Review by jsconan — APPROVED** [review]
It works well, together with #1814 which is a prerequisite.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**